tests/column_types/test_string.py

```diff
@@ -9,23 +9,23 @@
 
 
 def test_validate_min_length() -> None:
-    column = dy.String(min_length=2)
+    column = dy.String(min_length=2, nullable=True)
```

Comment:
I think I don't understand some of these changes -- we never have NULLs in the data, do we?